.github/workflows/main.yml

```diff
@@ -0,0 +1,149 @@
+#        run: echo "::set-output name=dir::$(yarn cache dir)"
+#
+#      - uses: actions/cache@v2
+#        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+#        with:
+#          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+#          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+#          restore-keys: |
+#            ${{ runner.os }}-yarn-
+#
+#      - name: Installing dependencies
+#        run: yarn install --frozen-lockfile
+#
+#      - name: Running tests
+#        run: yarn test
+#
+#  coverage:
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@v1
+#        with:
+#          fetch-depth: 1
+#
+#      - name: Setup Node.js
+#        uses: actions/setup-node@v1
+#        with:
+#          node-version: 16.9.1
+#
+#      - name: Get yarn cache directory path
+#        id: yarn-cache-dir-path
+#        run: echo "::set-output name=dir::$(yarn cache dir)"
+#
+#      - uses: actions/cache@v2
+#        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+#        with:
+#          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+#          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+#          restore-keys: |
+#            ${{ runner.os }}-yarn-
+#
+#      - name: Installing dependencies
+#        run: yarn install --frozen-lockfile
+#
+#      - name: Running coverage
+#        run: yarn coverage
+#
+#      - name: Report coverage
+#        uses: romeovs/lcov-reporter-action@v0.2.21
+#        with:
+#          lcov-file: ./coverage/lcov.info
+#          github-token: ${{ secrets.GITHUB_TOKEN }}
+#
+#      - name: Enforce coverage
+#        uses: VeryGoodOpenSource/very_good_coverage@v1.1.1
+#        with:
+#          path: ./coverage/lcov.info
+#          min_coverage: 80
+#
+#  deploy:
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@v1
+#        with:
+#          fetch-depth: 1
+#
+#      - name: Setup Node.js
+#        uses: actions/setup-node@v1
+#        with:
+#          node-version: 16.9.1
+#
+#      - name: Get yarn cache directory path
+#        id: yarn-cache-dir-path
+#        run: echo "::set-output name=dir::$(yarn cache dir)"
+#
+#      - uses: actions/cache@v2
+#        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+#        with:
+#          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+#          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+#          restore-keys: |
+#            ${{ runner.os }}-yarn-
+#
+#      - name: Installing dependencies
+#        run: yarn install --frozen-lockfile
+#
+#      - name: Deploy contracts
+#        run: yarn void:deploy
+#
+#      - name: Fork Deploy contracts from rinkeby
+#        run: yarn fork:deploy rinkeby
+#        env:
+#          ETH_NODE_URI_RINKEBY: ${{ secrets.ETH_NODE_URI_RINKEBY }}
```

Comment:
clean commented code ?

Reply:
I've kept them because they're the next steps to activate.